README.md

```diff
@@ -385,8 +385,9 @@ import {
 } from '@opentelemetry/sdk-trace-base';
 
 const provider = new BasicTracerProvider();
-provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
-trace.setGlobalTracerProvider(provider);
```

Comment:
Is this line `trace.setGlobalTracerProvider(provider);` not needed as well? How do we register the provider?